daemon/cluster/convert/network.go

```diff
@@ -8,6 +8,7 @@ import (
 	types "github.com/docker/docker/api/types/swarm"
 	netconst "github.com/docker/libnetwork/datastore"
 	swarmapi "github.com/docker/swarmkit/api"
+	allocator "github.com/docker/swarmkit/manager/allocator"
```

Comment:
Looks pretty straightforward to me https://github.com/moby/moby/blob/8daeeacfc1126eafb2b6d00e2dc6e2f5a6ed78cc/vendor/github.com/docker/swarmkit/manager/allocator/networkallocator/networkallocator.go#L968-L976, so agree on duplicating the function